.github/workflows/wheels.yml

```diff
@@ -184,6 +184,12 @@ jobs:
           pip install -r test_requirements.txt
           cd .. # Can't import numpy within numpy src directory
           python -c "import numpy; print(numpy.__version__); numpy.test();"
+
+      - name: Check README rendering for PyPI
+        run: |
+          python -mpip install twine
+          twine check dist/*
```

Comment:
How does one see the result of this test?

Reply:
Ahh, I see it is in the sdist run of the wheel builds, and [prints out](https://github.com/numpy/numpy/actions/runs/3371079753/jobs/5592820836#step:6:79)
```
Checking dist/numpy-1.24.0.dev0+1037.gb6bd4ddb3.tar.gz: PASSED
```

It runs [`twine check dist/*`](https://twine.readthedocs.io/en/stable/index.html#twine-check) which 
> Checks whether your distribution’s long description will render correctly on PyPI.